server/src/index.js

```diff
@@ -8,20 +8,21 @@ import db from './config/database';
 import Config from './config/config';
 import auth from './routes/auth';
 import Socket from './sockets/chess.socket';
-
-
-db.connectDb();
+import router from './router';
 
 const app = express();
 const server = http.createServer(app);
 const port = Config.server.SERVER_PORT;
 const { log } = console;
+
+db.connectDb();
 app.use(cors());
 app.use(helmet());
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
 app.use('/auth', auth);
-app.get('/api', (req, res) => res.json({ text: 'Online Chess Game!' }));
+app.use('/api', router);
```

Comment:
`/api` is a confusing name for en endpoint. Perhaps `/getRatings` `/getResult` etc could convey more meaning about this endpoint.

Reply:
if the router has somehing simillar as `/something`, we can directly call it as `api.chessgame.com/something` than calling `api.chessgame.com/api/something`. As api is more or less implied with or without the subdomain.

Reply:
Makes sense.